gradle/libs.versions.toml

```diff
@@ -89,6 +89,7 @@ edittext-pinentry = "com.alimuzaffar.lib:pinentryedittext:2.0.6"
 faker = "com.github.blocoio:faker:1.2.9"
 
 firebase-config = "com.google.firebase:firebase-config:21.2.0"
+firebase-analytics = "com.google.firebase:firebase-analytics-ktx:21.3.0"
 
```

Comment:
Shouldn't we also remove the mixpanel packages from the code?

Reply:
Oh, good catch. missed it